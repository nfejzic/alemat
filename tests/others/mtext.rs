use alemat::{elements::Text, MathMl};

#[test]
fn text() {
    let out = MathMl::with_content(Text::from("Hello world")).render();

    crate::snap_test!(out, name: "others_text");
}
