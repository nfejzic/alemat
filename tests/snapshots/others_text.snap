---
source: tests/others/mtext.rs
expression: input
---
<math>
  <mtext>
    Hello world
  </mtext>
</math>

